style.css

```diff
@@ -0,0 +1,178 @@
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    border-bottom: 1px solid lightgrey;
+    background-color: #ffffff;
+    box-shadow: 0 4px 10px -2px lightgray;
+}
+
+body {
+    background-color: #f8fafc;
+}
+
+.logos {
+    width: 20px;
+    height: 20px;
+    margin: 0.5em;
+}
+
+span {
+    color: #645cff;
+}
+
+.Your-tasks, .container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 5em;
+}
+
+fieldset {
+    width: 35em;
```

Comment:
If you want to set the paragraphs to percentages of the parent, you should use percentages in your css. Em's are a length unit relative to font size, not to it's parent width, or height. http://www.w3schools.com/cssref/css_units.asp